Rstudio-Proteomics/Dockerfile

```diff
@@ -12,7 +12,8 @@ LABEL software.version="1"
 LABEL about.summary="Rstudio-Proteomics is an adaptation from the original images provided by the Rocker-org. The container provided here contains a Rstudio server configurated with several packages dedicated to Proteomcis data analysis.Rstudio-Proteomics is an adaptation from the original images provided by the Rocker-org. The container provided here contains a Rstudio server configurated with several packages dedicated to Proteomcis data analysis."
```

Comment:
this one does not follow rule to get version directory (Rstudio-Proteomics/x.y.z/Dockerfile)